tempesta_fw/http_frame.c

```diff
@@ -0,0 +1,1017 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2014 NatSys Lab. (info@natsys-lab.com).
+ * Copyright (C) 2015-2019 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+#if DBG_HTTP_FRAME == 0
+#undef DEBUG
+#endif
+#include "lib/fsm.h"
+#include "lib/str.h"
+#include "procfs.h"
+#include "http.h"
+#include "http_frame.h"
+
+
+#define FRAME_CLI_MAGIC		"PRI * HTTP/2.0\r\n\r\nSM\r\n\r\n"
```

Comment:
Why `CLI`? Just curious. I would prefer names closer to the standard to make searching easier, e.g. `PREFACE`, but it's just a matter of taste and nothing more.

Reply:
`CLI` - since only the client sends this magic sequence in its preface during HTTP/2 connection establishing (https://tools.ietf.org/html/rfc7540#section-3.5). But I agree that `PREFACE` will also not be superfluous here.
Done.

Reply:
Done.